### PR TITLE
spython: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

File: recipes/spython/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9cef4c79f2cb1fcafa4555f8abd4ccfbd7d2ef3081a9b526beb027b5ff52fe77
+
+build:
+  number: 0
+  skip: True  # [py2k]
```
Comment (chrisburr) [inline recipes/spython/meta.yaml]: Can you make this a noarch package and change the requirements to `python >=3`?
```suggestion
  noarch: python
```
([Though the `setup.py` suggests Python 2 should be supported](https://github.com/singularityhub/singularity-cli/blob/master/setup.py#L106))

Comment (tschoonj) [reply recipes/spython/meta.yaml]: Fixed!

The documentation [mentions](https://singularityhub.github.io/singularity-cli/install) that python3 is required.

**Comment by tschoonj** [comment]
Thanks!

**Comment by vsoch** [comment]
hey @tschoonj where is the file with maintainers for spython? The staged-recipes "recipes" folder seems to go elsewhere... :eyes:

**Comment by chrisburr** [comment]
Everything ends up in a dedicated repository which the maintainers have write access to. For `spython` it at the bottom of `recipe/meta.yaml` in: https://github.com/conda-forge/spython-feedstock

**Comment by vsoch** [comment]
Great, thanks @chrisburr :)

**Comment by tschoonj** [comment]
@vsoch am adding you as maintainer now... sorry for the delay...

**Comment by vsoch** [comment]
It's okay! I figured I'd give it a go, I just opened a PR :point_right:  https://github.com/conda-forge/spython-feedstock/pull/7

I'm an excited smithy! And looking for reasons to call myself a smithy :)

One quick note is that the rendering instructions mentioned in the doc are 404 https://conda-forge.org/docs/conda_smithy.html#how-to-re-render

**Comment by tschoonj** [comment]
You were seconds faster than me opening the PR 😄 

Don't worry about the 404, the rendering instructions url is updated via the re-render.

**Pull request merged**